src/FileSystem.ts

```diff
@@ -0,0 +1,20 @@
+import RNFetchBlob from 'rn-fetch-blob';
+
+class FileSystemStatic {
+    public getDocumentDir = () => RNFetchBlob.fs.dirs.DocumentDir;
```

Comment:
Maybe rn-fetch-blob has the functionality that we use in react-native-fs?

Reply:
Yes, there is an overlap, rn-fetch-blob has filesystem functionality and react-native-fs also has download functionality :)